### PR TITLE
add hiring button to the end of blog posts

### PR DESCRIPTION
**Pull request by jukecraft**
I'm not happy with it being positioned so far on the right, but I remember what a pain it was to make the button on the front page work across devices and browsers, so I don't really want to touch the css.

### CODE REVIEW

**Comment by franziskas** [comment]
I'm not happy with it being positioned so far on the right, but I remember what a pain it was to make the button on the front page work across devices and browsers, so I don't really want to touch the css.

File: _layouts/post.html [hunk 1]
```diff
@@ -38,7 +38,9 @@
         <hr/>
         <p class="cross-post"><a href="{{ page.canonical.href }}">This post was cross-posted to {{ page.canonical.name }}.</a></p>
         {% endif %}
+      <a class="btn-hire btn-u-orange ls-s1" href="/careers/craftsman/">We are hiring!</a>
```
Comment (SamirTalwar) [inline _layouts/post.html]: Links are inline elements. Should it be in a `<p>`, `<div>` or similar so we don't have a mix of block and inline elements in the containing `<div>`?

**Comment by SamirTalwar** [comment]
Blog hiring button

This looks… massive. Can we shrink it down? And maybe lose the dead space on the right of the text?

Maybe it's not supposed to look like that. I tried it using the latest versions of Firefox and Chrome on Mac.

**Comment by jukecraft** [comment]
As I said in the first comment, I don't really want to mess with _this_ version of the button, as it took a while to get it working for the front page on different devices/browsers. This is a quick fix until someone is happy to make another button that fits better :)

**Pull request closed without merging**